docs/tutorials/state_histograms.ipynb

```diff
@@ -0,0 +1,293 @@
+{
```

Comment:
<p><code>Pass it's result</code>  -&gt; <code>Pass its result</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4024/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
done